.github/labeler.yml

```diff
@@ -1,8 +1,46 @@
 # https://github.com/actions/labeler/blob/main/README.md
 
+# When extending this, remember that in the PR, the labeler will run against
```

Comment:
> If this will be edited many times, we could just change the labeler 'on' trigger:

Can you explain more what this gives us?

> Also if the author manually disables a label, it will get added on next push.

Yes, that's a side-effect of this. We have to decide whether that's a problem or not :). @shantha-14  @carlescufi  can you comment your opinions?